### PR TITLE
added git installation

### PR DESCRIPTION
**Pull request by dee6600**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -115,11 +118,29 @@ def install_pycharm(self):
...
-        os.system("systemctl start sshd.service");
-        os.system("systemctl enable sshd.service");
-        print("OpenSSH installation completed");
-
+        os.system("dnf install -y openssh-server")
+        os.system("systemctl start sshd.service")
+        os.system("systemctl enable sshd.service")
+        print("OpenSSH installation completed")
+       
+    def install_git(self):
```
Comment (khomesh24) [inline main.py]: GIT is pre-installed in Fedora

File: main.py [hunk 2]
```diff
@@ -115,11 +118,29 @@ def install_pycharm(self):
...
+        print("Installing git....")
+        os.system("dnf -y install git")
+        print("git installation completed")
+        
+       
+    def install_atom(self):
+        print("Installing Atom")
+        os.system("rpm --import https://packagecloud.io/AtomEditor/atom/gpgkey")
+        os.system("rpm --import https://packagecloud.io/AtomEditor/atom/gpgkey")
+        os.system("dnf install atom")
```
Comment (khomesh24) [inline main.py]: It will ask -y for installation

File: main.py (hunk 1 above)
Comment (dee6600) [reply main.py]: okay np will remove it!!!

**Comment by dee6600** [comment]
all done! for now

File: main.py (hunk 2 above)
Comment (dee6600) [reply main.py]: added to some other programs as well

**Pull request merged**